apps/ledger-live-mobile/src/newArch/features/Web3Hub/screens/Web3HubMain/components/HorizontalList/index.tsx

```diff
@@ -1,30 +1,25 @@
-import React from "react";
-import { StyleSheet, View } from "react-native";
+import React, { useCallback } from "react";
+import { StyleSheet } from "react-native";
 import { FlashList } from "@shopify/flash-list";
-import { Flex, Text } from "@ledgerhq/native-ui";
+import { Box, Flex, InfiniteLoader, Text } from "@ledgerhq/native-ui";
 import { AppManifest } from "@ledgerhq/live-common/wallet-api/types";
-import LoadingIndicator from "../../../../components/ManifestsList/LoadingIndicator";
 import MinimalAppCard from "./MinimalAppCard";
 
 type Props = {
   title: string;
   isLoading: boolean;
   data: AppManifest[];
   onEndReached?: () => void;
-  extraData: { onPress: (manifest: AppManifest) => void };
+  extraData: (manifest: AppManifest) => void;
```

Comment:
Related to the other comment on this
```suggestion
  onPressItem: (manifest: AppManifest) => void;
```